javascript-forms/javascript-forms-quiz-notes.md

```diff
@@ -0,0 +1,55 @@
+# javascript-forms-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What event is fired when a user places their cursor in a form control?
+'focus'
+- What event is fired when a user's cursor leaves a form control?
+'blur'
+- What event is fired as a user changes the value of a form control?
+'input'
+- What event is fired when a user clicks the `"submit"` button within a `<form>`?
+'submit'
+- What does the `event.preventDefault()` method do?
+it tells user agent that if the event does not get explicitly handled, its default action should not be taken as it normally would be
+- What does submitting a form without `event.preventDefault()` do?
+the browser will reload automatically
```

Comment:
Aside from the browser reloading what does the browser attempt to do with the data submitted?